### ISSUE TITLE
SRW - configure plots to be more user-friendly

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#606]
Currently when small values are displayed on the axes of the plots, they are represented in the scientific format, e.g., `1.5e+1` which could be more attractively represented as `15`:
![plot](https://cloud.githubusercontent.com/assets/13209176/21857869/0017a1b4-d7f5-11e6-934c-3183995074e2.png)
I suggest the values on the axes should be regular decimals if they are between 0.001 and 1000 (1.0e-3 < x < 1.0e+3).

**Comment by robnagler** [comment]
Would %3.1g work?

**Comment by mrakitin** [comment]
I don't think it's suitable:
```py
>>> print '%3.1g' % 10
1e+01
```

**Comment by robnagler** [comment]
Maybe this:

```
>>> print '%.3g' % 1000
1e+03
>>> print '%.3g' % 100
100
```

Should try out with https://github.com/d3/d3-format

**Comment by mrakitin** [comment]
Fraction numbers may be a problem:
```py
>>> print '%.3g' % 0.01
0.01
>>> print '%.3g' % 0.001
0.001
>>> print '%.3g' % 0.0001
0.0001
>>> print '%.3g' % 0.00001
1e-05
```
but it looks more or less fine.

**Comment by mrakitin** [comment]
Fixed via 486a8f0.

**Comment by mrakitin** [comment]
Now it looks like:
![plot](https://cloud.githubusercontent.com/assets/13209176/22035802/4902841a-dcbf-11e6-9674-847a0d039d6a.png)
